requirements.txt

```diff
@@ -1,2 +1,3 @@
 -e git+https://github.com/CodeForAfricaLabs/HURUmap.git@develop#egg=hurumap
 gunicorn==19.8.1
+-e git+https://github.com/CodeForAfricaLabs/wazimap.git@master#egg=wazimap
```

Comment:
@phillipahereza Won't this conflict with whatever is installed with https://github.com/CodeForAfricaLabs/HURUmap and we should therefore make this change there instead of here?